src/components/pages/Attendance/attendance.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const Attendance = () => {
+  return (
+    <div>
```

Comment:
I love here that you're using good practices in terms of stubbing out work that will continue to be worked on. 

The only thing I would like to change is the addition of a `div` that seemingly isn't needed. I would a fragment instead `<>`.

Reply:
Thanks for the feedback Ryan.  I just updated the attendance.js to reflect your change recommendations.